_api-reference/snapshots/get-snapshot-status.md

```diff
@@ -23,24 +23,29 @@ Path parameters are optional.
 | Parameter | Data type | Description | 
 :--- | :--- | :---
 | repository | String | Repository containing the snapshot. |
-| snapshot | String | Snapshot to return. |
+| snapshot | List | Snapshot(s) to return. |
+| index | List | Index/Indices to include in the response. |
 
 Three request variants provide flexibility:
 
 * `GET _snapshot/_status` returns the status of all currently running snapshots in all repositories.
 
 * `GET _snapshot/<repository>/_status` returns all currently running snapshots in the specified repository. This is the preferred variant.
 
-* `GET _snapshot/<repository>/<snapshot>/_status` returns detailed status information for a specific snapshot in the specified repository, regardless of whether it's currently running or not. 
+* `GET _snapshot/<repository>/<snapshot>/_status` returns detailed status information for a specific snapshot(s) in the specified repository, regardless of whether it's currently running or not. 
 
-Using the API to return state for other than currently running snapshots can be very costly for (1) machine machine resources and (2) processing time if running in the cloud. For each snapshot, each request causes file reads from all a snapshot's shards. 
+* `GET /_snapshot/<repository>/<snapshot>/<index>/_status` returns detailed status information only for the specified indices in a specific snapshot in the specified repository, for completed snapshots. Note that this variant works only for indices of a specific snapshot.
+
+Using the API to return state for other than currently running snapshots can be very costly for (1) machine machine resources and (2) processing time if running in the cloud. For each snapshot, each request causes file reads from all of the snapshot's shards. 
```

Comment:
@ltaragi: How would a user avoid returning states on snapshots that aren't currently running?

Reply:
Sorry did not fully get the question. The user can make use of the `snapshot.max_shards_allowed_in_status_api` cluster setting or indexes-filter to limit the cost of returning states for completed snapshots.